compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -192,10 +192,7 @@ impl<'a> IntoIterator for LLVMFeature<'a> {
 // to LLVM or the feature detection code will walk past the end of the feature
 // array, leading to crashes.
 //
-// To find a list of LLVM's names, check llvm-project/llvm/include/llvm/Support/*TargetParser.def
-// where the * matches the architecture's name
-//
-// For targets not present in the above location, see llvm-project/llvm/lib/Target/{ARCH}/*.td
+// To find a list of LLVM's names, see llvm-project/llvm/lib/Target/{ARCH}/*.td
```

Comment:
Considering the "Beware to not use the LLVM github project for this, but check the git submodule" warning later, could this segment just be, like... a hyperlink?

Reply:
My concern there would be how to write that link, given that it would need to refer to whatever commit in rust-llvm is currently in use

Reply:
GitHub does not allow me to highlight lines that you did not diff, but I am talking about the context of the next 10 or so lines, **not** this part. The list of instructions that follows includes telling people a step-by-step process on how to find the repository, as if landmarks are all we have. The link does NOT have to be exact, we can start with a link to the repository.

Reply:
Ah got it. Will do